### DISCUSSION TITLE
Maybe new feature: auto-quick

### DISCUSSION THREAD

**Discussion by jsirex** [minad/corfu#422] [Ideas]
Unfortunately, my lisp programming skills are basic, so I don't know proper way how to implement this, so can't create a PR.

Completion with quick key is really handy, but:

is it possible to make quick (or may be another extension) automatically enabled?

Scenario: when I type something, I see list and I see my candidate. Currently I have to press "M-q" then look for key (let it be f), then press f.
Improved scenario: when I type something I see the list **and quick keys for each item**. I can either continue typing or press, for example "M-<THE-QUICK-KEY>" to insert/complete my candidate. 
Same way currently works company mode, but they use number. This scenario happens often: you type something, ending up with couple of candidate and usually yours is second or third".
And, if there are too many candidates, just limit and have quick keys only for first 10-20.

Same behavior probably will be good for vertico.

**Comment by minad** [comment]
Hi, this has been proposed before, e.g. in https://github.com/minad/corfu/issues/231. It is planned to add this at some point.
